lib/linter/linter.js

```diff
@@ -307,10 +307,6 @@ function getDirectiveComments(filename, ast, ruleMapper, warnInlineConfig) {
         const directiveText = match[1];
         const lineCommentSupported = /^eslint-disable-(next-)?line$/u.test(directiveText);
```

Comment:
This variable name will be quite confusing after this change. Thoughts about renaming it to something like `mustBeOnSingleLine`?